<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PartnerEditDialog</class>
 <widget class="QDialog" name="PartnerEditDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Партнёр</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">

   <item>
    <layout class="QFormLayout" name="formLayout">
     <!-- Наименование -->
     <item row="0" column="0">
      <widget class="QLabel" name="labelName">
       <property name="text">
        <string>Наименование:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineName"/>
     </item>

     <!-- Тип партнёра -->
     <item row="1" column="0">
      <widget class="QLabel" name="labelType">
       <property name="text">
        <string>Тип партнёра:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="comboType"/>
     </item>

     <!-- Рейтинг -->
     <item row="2" column="0">
      <widget class="QLabel" name="labelRating">
       <property name="text">
        <string>Рейтинг:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="spinRating">
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>

     <!-- Директор -->
     <item row="3" column="0">
      <widget class="QLabel" name="labelDirector">
       <property name="text">
        <string>ФИО директора:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="lineDirector"/>
     </item>

     <!-- Телефон -->
     <item row="4" column="0">
      <widget class="QLabel" name="labelPhone">
       <property name="text">
        <string>Телефон:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="linePhone"/>
     </item>

     <!-- Email -->
     <item row="5" column="0">
      <widget class="QLabel" name="labelEmail">
       <property name="text">
        <string>Email:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="lineEmail"/>
     </item>

     <!-- ИНН -->
     <item row="6" column="0">
      <widget class="QLabel" name="labelInn">
       <property name="text">
        <string>ИНН:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="lineInn"/>
     </item>

     <!-- Адрес -->
     <item row="7" column="0">
      <widget class="QLabel" name="labelAddress">
       <property name="text">
        <string>Адрес:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QTextEdit" name="textAddress"/>
     </item>

    </layout>
   </item>

   <!-- Кнопки OK / Cancel -->
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>

  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PartnerEditDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>404</y>
    </hint>
    <hint type="destinationlabel">
     <x>248</x>
     <y>224</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PartnerEditDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>404</y>
    </hint>
    <hint type="destinationlabel">
     <x>316</x>
     <y>224</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>